config/examples/AnyCubic/i3 Mega/Trigorilla AVR/Configuration.h

```diff
@@ -1736,7 +1745,7 @@
 #define Y_BED_SIZE 210
 
 // Travel limits (linear=mm, rotational=°) after homing, corresponding to endstop positions.
-#define X_MIN_POS -5
+#define X_MIN_POS 0
 #define Y_MIN_POS 0
 #define Z_MIN_POS 0
 #define X_MAX_POS X_BED_SIZE
```

Comment:
Is it true that this is the farthest to the right the X carriage can move to?

If your aim is to get the center coordinates 105, 105 to line up with the center of the bed, you can tweak other values and keep `X_MIN_POS` at -5 if the nozzle is left of the bed when homed.

Reply:
I'll be honest I didn't check the centre position. I was mostly concerned about getting the left edge of bed usable.

Reply:
Thanks for letting me double check. Mine seems to have an optional bed that's bigger than most.